test/onnx/gen_onnx.py

```diff
@@ -7157,6 +7157,42 @@ def split_test_no_attribute():
     return ([const_node, node], [x], [y1, y2, y3, y4])
 
 
+@onnx_test()
+def split_test_uneven():
+    x = helper.make_tensor_value_info('x', TensorProto.FLOAT, [12, 15])
+    y1 = helper.make_tensor_value_info('y1', TensorProto.FLOAT, [3, 15])
+    y2 = helper.make_tensor_value_info('y2', TensorProto.FLOAT, [3, 15])
+    y3 = helper.make_tensor_value_info('y3', TensorProto.FLOAT, [2, 15])
+    y4 = helper.make_tensor_value_info('y4', TensorProto.FLOAT, [2, 15])
+    y5 = helper.make_tensor_value_info('y5', TensorProto.FLOAT, [2, 15])
```

Comment:
This test highlights the possible issue

Reply:
In this particular case, if we assumed that "the last chunk" from the ONNX spec means the last output, then the only way to split this into 5 outputs would be `[3, 3, 3, 3, 0]`. I'm not sure if it makes sense to leave the last output empty.

Reply:
https://github.com/onnx/onnx/pull/4481

https://github.com/onnx/onnx/blob/60b0a34037f2980c519793b1bbc3a6ed79b0dfa3/onnx/defs/tensor/defs.cc#L779

Need to mimic this logic.